accesstoken2/src/app.ts

```diff
@@ -72,17 +54,28 @@ export class AccessTokenLambda implements LambdaInterface {
                 statusCode: 200,
                 body: JSON.stringify(accessTokenResponse),
             };
-        } catch (err: any) { //Todo dont want any
+        } catch (err: unknown) { //Todo dont want any
+            for (const appError of ApplicationErrors) {
```

Comment:
I suspect we can use some Typescript magic to make this for of loop redundant

Reply:
Yeah actually I didn't need I had two ideas in my head and implemented both